assets/large_language_models/rag/components/src/flow_creation.py

```diff
@@ -174,13 +255,17 @@ def main(args, ws, current_run, activity_logger: Logger):
     print("embedding_deployment_name: %s" % embedding_deployment_name)
     print("embedding_model_name: %s" % embedding_model_name)
 
+    # Hard coded for ease of reverting if there are issues
+    code_first = True
```

Comment:
Move it to the top of the file, along with other constants

Reply:
moved